core/src/validations.rs

```diff
@@ -1,13 +1,29 @@
-use witnet_data_structures::chain::Block;
+use witnet_crypto::hash::Sha256;
+use witnet_crypto::merkle::merkle_tree_root;
+use witnet_data_structures::chain::{Block, Hash, Hashable, Transaction};
 
 /// Function to validate block's coinbase
 pub fn validate_coinbase(_block: &Block) -> bool {
     // TODO Implement validate coinbase algorithm
     true
 }
 
+/// Function to calculate a merkle tree from a transaction vector
+pub fn create_merkle_tree(transactions: &[Transaction]) -> Hash {
```

Comment:
_Suggestion_
I don't like the name `create_merkle_tree` too much since is basically the same function as `merkle_tree_root` but lifted to work with Transaction and Hash instead of Sha256. I would just call it `merkle_tree_root` to be consistent with the one in `witnet_crypto::merkle`.